xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -8,132 +8,146 @@
      from Swift</literal>.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Launch the restore.
+     Launch the restore process with <literal>freezer job-start</literal>
+     <replaceable>JOB-ID</replaceable>.
     </para>
-<screen>freezer-scheduler -c &lt;hostname&gt; job-start -j &lt;job-id&gt;</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     This will take some time. While you wait you can follow the progress in
-     the <literal>/var/log/freezer-agent/freezer-scheduler.log</literal> log
-     file.
+     This will take some time. You can follow the progress in
+     <literal>/var/log/freezer/freezer-scheduler.log</literal>.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Log in to the &lcm;.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Stop the Percona DB service:
+     Log back in to the first node running the &mariadb; service, the same node
+     as in <xref linkend="mariadb-nodes"/>.
     </para>
-<screen>cd ~/scratch/ansible/next/ardana/ansible
-ansible-playbook -i hosts/verb_hosts percona-stop.yml</screen>
-   </listitem>
-   <listitem>
-    <para>
-     Log back in to the first node running the &mariadb; service. (Same node as
-     step #3)
-    </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Clean the &mariadb; directory using this command:
     </para>
 <screen>sudo rm -r /var/lib/mysql/*</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Copy the restored files back to the &mariadb; directory:
     </para>
 <screen>sudo cp -pr /tmp/mysql_restore/* /var/lib/mysql</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Log in to each of the other nodes in your &mariadb; cluster (the other nodes
-     determined from step #2) and remove the <literal>grastate.dat</literal>
-     file from each of them:
+     Log in to each of the other nodes in your &mariadb; cluster, which were
+     determined in <xref linkend="mariadb-nodes"/>. Remove the
+     <literal>grastate.dat</literal> file from each of them.
     </para>
 <screen>sudo rm /var/lib/mysql/grastate.dat</screen>
-   <note>
-    <para>
-     Do not remove this file from the first node in your &mariadb; cluster.
-     Ensure you only do this from the other cluster nodes.
-    </para>
-   </note>
-   </listitem>
-   <listitem>
+    <warning>
+     <para>
+      Do not remove this file from the first node in your &mariadb; cluster.
+      Ensure you only do this from the other cluster nodes.
+     </para>
+    </warning>
+   </step>
+   <step>
     <para>
      Log back in to the &lcm;.
     </para>
-   </listitem>
-   <listitem>
-    <para>
-     Start the Percona DB service back up:
-    </para>
-<screen>cd ~/scratch/ansible/next/ardana/ansible
-ansible-playbook -i hosts/verb_hosts galera-bootstrap.yml</screen>
```

Comment:
please check operationswiftfix3 in the jira to add this step back ,but call it start mariaDB service